src/Tribe/Aggregator/Record/Abstract.php

```diff
@@ -2606,7 +2604,7 @@ public function get_last_child_post( $force = false ) {
 			return $this->last_child;
 		}
 
-		$children_query_args = array( 'posts_per_page' => 1, 'order' => 'DESC', 'order_by' => 'modified' );
+		$children_query_args = [ 'posts_per_page' => 1, 'order' => 'DESC', 'order_by' => 'modified' ];
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 6bc88816c138f790e5aefb9570b7e1f1</sub>